src/content/manifest/overview-manifest.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Okteto Manifest
+description: An overview of the Okteto Manifest and its core features.
+sidebar_label: Overview
+id: overview-manifest
+---
+
+The Okteto manifest enables you to provide configuration for building, deploying, and developing your application in a Development Environment. This file is optional if you already have a Docker Compose file for your application. Learn more about the Okteto Manifest in our reference section [here](reference/okteto-manifest.mdx).
+
+### Sample Okteto Manifest
+
+```yaml
+icon: https://apps.okteto.com/movies/icon.png
+build:
+  api:
+    context: .
+deploy:
+  - helm upgrade --install app chart --set image=${OKTETO_BUILD_API_IMAGE}
+dev:
+  api:
+    command: ["bash"]
+    forward:
+      - 8080:8080
+      - 9229:9229
+    sync:
+      - .:/usr/src/app
+```
+
+This example shows how to [build](reference/okteto-manifest.mdx/#build-object-optional) the container image of the **api** microservice, how to [deploy](reference/okteto-manifest.mdx/#deploy-object-optional) the application using its helm chart,
+and how to [develop](reference/okteto-manifest.mdx/#dev-object-optional) on the **api** microservice by synchronizing the local filesystem on the container path `/usr/src/app`.
+If you want to learn more about what all you can configure in this manifest, head over to the [Okteto Manifest Reference](reference/okteto-manifest.mdx).
+
+### Built-in Environment Variables
```

Comment:
this is dup in both content, is it on purpose?

Reply:
Nope, not on purpose and I think fixed now? Or at least I'm not seeing the duplicate content anymore